var/spack/repos/builtin/packages/hssp/package.py

```diff
@@ -6,12 +6,12 @@
 from spack import *
 
 
-class Xssp(AutotoolsPackage):
+class Hssp(AutotoolsPackage):
```

Comment:
Can you also update this docstring, similarly to what was done for the other package?

Reply:
added this already, btw